lib/api/endpoints/experiment.rb

```diff
@@ -59,7 +60,12 @@ class Experiment < Grape::API
       end
       delete '/:id', authorize: [:write, ::Experiment] do
         status 204
-        ::Experiment.delete(declared_params[:id])
+
+        experiment = ::Experiment.find(declared_params[:id])
+        Events::PostgresSink.call(experiment)
```

Comment:
It turns out I can't just have a universal document update, and that [they distinguish](https://github.com/elastic/elasticsearch-rails/tree/master/elasticsearch-model#custom-callbacks) between different CRUD events. Fuck. 

OK, at least it's an easy fix.